tests/Service/FormatSponsorshipLevelBenefitTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: btaralle
+ * Date: 2019-02-07
+ * Time: 17:41
+ */
+
+namespace App\Tests\Service;
+
+use App\Repository\SponsorshipBenefit\SponsorshipBenefitManagerInterface;
+use App\Repository\SponsorshipLevel\SponsorshipLevelManagerInterface;
+use App\Repository\SponsorshipLevelBenefit\SponsorshipLevelBenefitManagerInterface;
+use App\Service\FormatSponsorshipLevelBenefit;
+use Prophecy\Argument;
+use PHPUnit\Framework\TestCase;
+
+class FormatSponsorshipLevelBenefitTest extends TestCase
+{
+    private $sponsorshipBenefitManagerProphecy;
+    private $sponsorshipLevelManagerProphecy;
+    private $sponsorshipLevelBenefitManagerProphecy;
+    private $formatSponsorshipLevelBenefit;
+
+    public function setUp()
+    {
+        $this->sponsorshipBenefitManagerProphecy = $this->prophesize(SponsorshipBenefitManagerInterface::class);
+        $this->sponsorshipLevelManagerProphecy = $this->prophesize(SponsorshipLevelManagerInterface::class);
+        $this->sponsorshipLevelBenefitManagerProphecy = $this->prophesize(SponsorshipLevelBenefitManagerInterface::class);
+
+        $this->formatSponsorshipLevelBenefit = new FormatSponsorshipLevelBenefit(
+            $this->sponsorshipBenefitManagerProphecy->reveal(),
+            $this->sponsorshipLevelManagerProphecy->reveal(),
+            $this->sponsorshipLevelBenefitManagerProphecy->reveal()
+        );
+    }
+
+    public function testFormat()
```

Comment:
We haven't discussed it yet, but I usually use snake case for tests name.

Opinions @Diwoka @kevinjhappy @lbassin

Reply:
@tdutrion for me here need to adopt KISS principle a class test it a class after all and change the way to write method in a class is not good way to simplify code